src/backend/db/products.js

```diff
@@ -5,26 +5,111 @@ import { v4 as uuid } from "uuid";
  * You can add products of your wish with different attributes
  * */
 
+const url = "assets/images/product/";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    name: "Breadboard Circuit Test Board",
+    price: 65,
+    img: `${url}breadboard1.png`,
+    category: "Breadboard",
   },
   {
     _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    name: " Printed Circuit Board",
+    price: 285,
+    img: `${url}pcb1.png`,
+    category: "Breadboard",
   },
   {
     _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
+    name: "Mini Breadboard",
+    price: 29,
+    img: `${url}minibreadb.png`,
```

Comment:
Suggestion to host the images live links on image hosting sites such As ImageKit or Cloudinary so that the file size of your app gets reduced!!